### PR TITLE
fix(types): do not publish built/globals.d.ts

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by cnishina** [comment]
Breaks things in bad ways... needs investigation.

File: lib/browser.ts [hunk 1]
```diff
@@ -679,8 +679,12 @@ export class ProtractorBrowser extends Webdriver {
                     let $timeout = $delegate;
 
                     let taskId = 0;
-                    if (!window['NG_PENDING_TIMEOUTS']) {
-                      window['NG_PENDING_TIMEOUTS'] = {};
+
```
Comment (juliemr) [inline lib/browser.ts]: Thanks for fixing here - we should actually just move this whole function into `clientSideScripts` so we don't deal with TS here at all. 
I filed an issue at https://github.com/angular/protractor/issues/3550

File: lib/config.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+export interface HttpProxyAgent { constructor(opts: Object): HttpProxyAgent; }
```
Comment (juliemr) [inline lib/config.ts]: Why not just type that as `any`? I'm not sure why this particular one requires a special type.

File: lib/browser.ts (hunk 1 above)
Comment (cnishina) [inline lib/browser.ts]: This was really messy and I agree we should pull this out of `browser.ts`

File: lib/config.ts (hunk 2 above)
Comment (cnishina) [inline lib/config.ts]: I agree, we should remove the type in the config. `HttpProxyAgent` does not have an ambient typings; however, we should not have to define the interface for the constructor.

**Comment by juliemr** [comment]
Passes all my local tests, the TS looks good. Let's merge!

**Pull request merged**